api/config.py

```diff
@@ -47,6 +47,9 @@
         'db_server_selection_timeout': '3000',
         'data_path': os.path.join(os.path.dirname(__file__), '../persistent/data'),
     },
+    'elasticsearch': {
+        'es_endpoint': 'flywheel-elasticsearch:9200'
```

Comment:
if this can be configured outside of the core module, I am happy to remove it from here.